pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -487,6 +487,13 @@ The delayed message index time step(in seconds) in per bucket snapshot segment,
     )
     private long brokerShutdownTimeoutMs = 60000;
 
+    @FieldContext(
+            category = CATEGORY_SERVER,
+            dynamic = true,
```

Comment:
dynamic = true

Is it necessary? And I don't see a way to modify it dynamically

Reply:
Same as above

Reply:
So how do you dynamically change this parameter? You are not even listening to the metastore

Reply:
@315157973 @labuladong 

Maybe modifying this api like below is better?
- How many bundles will be unloaded every `{num}` seconds
- How many seconds do all the bundles at the same speed